pkgs/by-name/m1/m1-terraform-provider-helper/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule {
+  pname = "m1-terraform-provider-helper";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "kreuzwerker";
+    repo = "m1-terraform-provider-helper";
+    rev = "0.9.0";
+    sha256 = "0plng7kz768v1km8mxw7jzsrvpxfzg991lrwwndpkgnyk2m28cnv";
+  };
+
+  # Only build from the root package
+  subPackages = [ "." ];
+
+  vendorHash = "sha256-+UzPs1kiapYy5WMwx6EkQBlBs3DpWhzaQcYqdlzH94U=";
+
+  # Use custom build command from Makefile
+  buildPhase = ''
+    make build-release VERSION=$version
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp dist/release/m1-terraform-provider-helper $out/bin/
+  '';
+
```

Comment:
Consider adding `passthru.updateScript = nix-update-script { };` as it will allow automatic update PRs by the bot when new releases ahppen.

Reply:
Added